crates/matrix-sdk-crypto/src/backups/keys/recovery.rs

```diff
@@ -203,13 +203,19 @@ impl RecoveryKey {
         let message = Message::from_base64(ciphertext, mac, ephemeral_key)?;
         let pk = self.get_pk_decrytpion();
 
-        pk.decrypt(&message).map(encode)
+        let decrypted = pk.decrypt(&message)?;
+
+        Ok(String::from_utf8_lossy(&decrypted).to_string())
```

Comment:
Why `_lossy`?

Reply:
So we don't have to deal with the error, I think I'll want to propagate the `Vec<u8>` based encryption/decryption API, now that we're not tied to strings because of `libolm`, just not right now.